manifests/m/MetaCubeX/mihomo/1.19.1/MetaCubeX.mihomo.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=NVS1.LF.7-5-0.Unix
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: MetaCubeX.mihomo
+PackageVersion: 1.19.1
+PackageLocale: en-US
+Publisher: MetaCubeX
+PublisherUrl: https://github.com/MetaCubeX/mihomo/releases
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: MetaCubeX
+PackageName: Mihomo
+PackageUrl: https://github.com/MetaCubeX/mihomo
+License: MIT
+LicenseUrl: https://github.com/MetaCubeX/mihomo?tab=MIT-1-ov-file
+Copyright: MetaCubeX
+# CopyrightUrl:
+ShortDescription: 'A simple python pydantic model (type hint and autocompletion support) for Honkai: Star Rail parsed data from the Mihomo API.'
+# Description:
+Moniker: clash
```

Comment:
I don't understand how `clash` fits as a moniker or as the tags. Can you provide some additional information or documentation showing that `clash` is a commonly used alternative name for this application?

Reply:
This is an incorrect configuration line. Historically, it was named "clash-meta." To avoid security issues, they renamed it to "mihomo."